dts/bindings/sensor/ti,fdc2x1x.yaml

```diff
@@ -0,0 +1,235 @@
+
+        0 = Perform continuous conversions on Channel 0
+        1 = Perform continuous conversions on Channel 1
+        2 = Perform continuous conversions on Channel 2 (FDC2114, FDC2214 only)
+        3 = Perform continuous conversions on Channel 3 (FDC2114, FDC2214 only)
+      enum:
+        - 0
+        - 1
+        - 2
+        - 3
+
+    deglitch:
+      type: int
+      required: false
+      default: 5
+      description: |
+        Input deglitch filter bandwidth. Select the lowest setting that exceeds
+        the oscillation tank oscillation frequency.
+
+        1 = 1MHz
+        4 = 3.3MHz
+        5 = 10MHz
+        7 = 33MHz
+      enum:
+        - 1
+        - 4
+        - 5
+        - 7
+
+    sensor-activate-sel:
+      type: string
+      required: false
+      default: "low-power"
+      description: |
+        Sensor Activation Mode Selection. Set the mode for sensor initialization.
+
+        full-current = the FDC will drive maximum
+        sensor current for a shorter sensor activation time.
+
+        low-power = the FDC uses the value
+        programmed by chx_idrive during sensor activation to minimize
+        power consumption.
+      enum:
+        - "full-current"
+        - "low-power"
+
+    ref-clk-src:
+      type: string
+      required: false
+      default: "internal"
+      description: |
+        Select Reference Frequency Source.
+
+        internal = Use Internal oscillator as reference frequency
+        external = Reference frequency is provided from CLKIN pin.
+      enum:
+        - "internal"
+        - "external"
+
+    current-drive:
+      type: string
+      required: false
+      default: "normal"
+      description: |
+        Select Current Sensor Drive
+
+        normal = The FDC will drive all channels with normal sensor current
+        (1.5mA max).
+
+        high = The FDC will drive channel 0 with current >1.5mA.
+
+        This mode is not supported if autoscan = "disabled" (multichannel mode)
+      enum:
+        - "normal"
+        - "high"
+
+    output-gain:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Output gain control (FDC2112, FDC2114 only)
+
+        0 = Gain = 1   | Effective Resolution 12 bits | 100% full scale
+        1 = Gain = 4   | Effective Resolution 14 bits | 25% full scale
+        2 = Gain = 8   | Effective Resolution 15 bits | 12.5% full scale
+        3 = Gain = 16  | Effective Resolution 16 bits | 6.25% full scale
+      enum:
+        - 0
+        - 1
+        - 2
+        - 3
+
+child-binding:
+  description: |
+    Settings for each channel 0-1 (FDC2112 and FDC2212) or
+    0-3 (FDC2114 and FDC2214)
+
+  properties:
+      rcount:
+        type: int
+        required: false
+        default: 7499
+        description: |
+          Channel X Reference Count Conversion Interval Time.
+          Valid range: 256 - 65535
+
+      offset:
+        type: int
+        required: false
+        default: 0
```

Comment:
Same comments about default made above apply here also.

Reply:
This value applies only to the 12 Bit version. So I dont think it makes sense to make `required`. I leave the default power-on-reset value as `default`